### DISCUSSION TITLE
cannot query hive table from hive catalog

### DISCUSSION THREAD

**Discussion by richetgeek** [StarRocks/starrocks#18395] [Announcements]
1.create hive catalog
CREATE EXTERNAL CATALOG hive PROPERTIES (
    'type'='hive',
    'hive.metastore.uris' = 'thrift://xxxxx:9083',
    'hadoop.username' = 'hive',
    'dfs.nameservices'='xxxx',
    'dfs.ha.namenodes.bjCluster'='nn1,nn2',
    'dfs.namenode.rpc-address.bjCluster.nn1'='xxx',
    'dfs.namenode.rpc-address.bjCluster.nn2'='xxxx',
    'dfs.client.failover.proxy.provider.bjCluster'='org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider'
);
2.copy hdfs-site.xml, core-site.xml to be/conf,fe/conf 
3.restart fe.be
table meta can show ,but query timiout 300s,not err lg

**Comment by leoyy0316** [comment]
try use limit 10, if ok, you can set query_timeout, set query_timeout = 600;
